### ISSUE TITLE
CPU usage 100% in Linux (ubuntu 20.04)

### ISSUE THREAD

**Issue by sakya** [AvaloniaUI/Avalonia#5192]
**CPU usage 100% in Linux (ubuntu 20.04)**
The project XamlControlsGallery under Ubuntu 20.04 and using Avalonia 0.10.0-rc1 has a CPU usage of 100%

**To Reproduce**
1. Download the XamlControlsGallery source code 
2. Update reference to Avalonia 0.10.0-rc1
3. Run

**Expected behavior**
The app shouldn't use 100% of CPU

**Screenshots**
![image](https://user-images.githubusercontent.com/289552/102688295-31e84d00-41f6-11eb-9ebb-82db2e0ab976.png)

![image](https://user-images.githubusercontent.com/289552/102688272-036a7200-41f6-11eb-8ce0-98bb68025909.png)

**Desktop:**
 - OS: Linux Ubuntu 20.04
 - Version 0.10.0-rc1

**Comment by danwalmsley** [comment]
Does your system have a gpu or is it relying on software rendering?

**Comment by sakya** [comment]
I'm using a GeForce GTX 760M (Xorg)
![image](https://user-images.githubusercontent.com/289552/102692162-d3c96300-4211-11eb-806c-08fb182d8394.png)

**Comment by sakya** [comment]
Driver version 
`NVIDIA-SMI 450.80.02    Driver Version: 450.80.02    CUDA Version: 11.0`

**Comment by devel0** [comment]
I experienced same problem using 0.10.0-preview5 and nvidia gtx 1050ti but using latest **0.10.0-rc1** problem solved.

**Comment by sakya** [comment]
Hi, I cleaned the build, changed to net5.0 but the cpu is always 100%
This seems really related to Linux because the same project run on Windows on a Celeron J1900 with Intel integrated graphics uses 2/5% of  CPU.

[XamlControlsGallery-master.zip](https://github.com/AvaloniaUI/Avalonia/files/5742873/XamlControlsGallery-master.zip)

**Comment by devel0** [comment]
Thanks, It's ok its related to known bug ( https://github.com/AvaloniaUI/Avalonia/issues/2012 ) if you comment out this cpu goes ok

```xml
      <!-- <ProgressBar IsIndeterminate="True" Margin="0 40" /> -->
```

btw for my project where I not used progress bar I had that problem of cpu when using the previous 0.10.0-preview5 while 0.10.0-rc1 solve, so seems there are some progress fixating the issue of high cpu usage on x11 excluding in fact the progressbar when indeterminate mode.

**Comment by sakya** [comment]
You're right, many thanks!
I had this problem with a different project. In my project I don't have any indeterminate progress bar but I was able to fix the issue.
I have a spinner with an animation set to INFINITE. The spinner is shown or hidden...well when it's hidden it seems that the animation runs and uses 100% of the CPU.
I solved the issue by stopping and starting the animation when i show/hide the spinner.
So it seems that on Linux animations run even on hidden elements, while on Windows not?
